### PR TITLE
Fixing broken test

### PR DESCRIPTION
**Pull request by vicmaster**
- Removing deprecated warnings
- Fixing broken test
- Cleaning spec_helper

### CODE REVIEW

File: app/models/cfp/proposal.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Proposal < ActiveRecord::Base
     belongs_to :user, class_name: "::User"
     has_many :comments
     has_many :ranks
-    scope :approved, where(:approved => true)
+    scope :approved, -> { where(:approved => true) }
```
Comment (alexesba) [inline app/models/cfp/proposal.rb]: I think you can use the new  syntax  for the hashes:
scope :approved, -> { where( approved: true) }

Comment (vicmaster) [inline app/models/cfp/proposal.rb]: Oh! I see, let me make a change

File: app/views/cfp/proposals/_proposal.html.haml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     %td
       = link_to t('actions.edit'), edit_proposal_path(proposal)
     %td
-      = link_to t('actions.delete'), proposal_path(proposal), :method => :delete, :confirm => t('actions.sure')
+      = link_to t('actions.delete'), proposal_path(proposal), :method => :delete, :data => t('actions.sure')
```
Comment (alexesba) [inline app/views/cfp/proposals/_proposal.html.haml]: same here: 
 = link_to t('actions.delete'), proposal_path(proposal), method: :delete, data:  t('actions.sure')

**Comment by alexesba** [comment]
Beside my comments:  LGTM

**Pull request merged**